presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -153,6 +153,25 @@ Mathematical Functions
     array ``bins``. The ``bins`` parameter must be an array of doubles and is
     assumed to be in sorted ascending order.
 
+Statistical Functions
+-----------------------
+
+.. function:: wilson_interval_lower(pos, neg) -> double
```

Comment:
Use [, z] for optional parameter. Otherwise, each version should have a separate description.